diagnostic.md

```diff
@@ -45,7 +58,10 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+  $( "*" ).find( "class" ).css( "big" );
```

Comment:
This isn't the jQuery diagnostic! We're just looking for CSS selectors. 
.big
#contentPane
.important.red
div > p